mackup/applications/lftp.cfg

```diff
@@ -0,0 +1,8 @@
+[application]
+name = lftp
+
+[configuration_files]
+rc
+
+[xdg_configuration_files]
+.config/lftp
```

Comment:
You don't have to specify the XDG folder, I suggest trying your conf until it works before opening a PR.

Thanks!